torch/csrc/jit/codegen/cuda/ops/normalization.h

```diff
@@ -43,6 +46,89 @@ struct VarMeanResult {
   TensorView* mean = nullptr;
 };
 
+} // namespace cuda
+} // namespace fuser
+} // namespace jit
+} // namespace torch
+
+namespace std {
+
+// Make these results behave like a std::tuple
```

Comment:
This behavior makes the implementation of `cpp-repro-gen.py` easier, and I believe this is also generally helpful.